tests/probes_test.go

```diff
@@ -286,3 +270,21 @@ func createProbeSpecification(period int32, initialSeconds int32, handler v12.Ha
 func isHTTPProbe(probe v12.Probe) bool {
 	return probe.Handler.HTTPGet != nil
 }
+
+func serverStarter(vmi *v12.VirtualMachineInstance, probe *v12.Probe, port int) {
+	if isHTTPProbe(*probe) {
+		tests.StartHTTPServer(vmi, port)
+	} else {
+		tests.StartTCPServer(vmi, port)
+	}
+}
+
+func attachProbeToPodByIp(pod *v1.Pod, IPFamily v1.IPFamily, probe *v12.Probe) (*v12.Probe, error) {
```

Comment:
This name is not OK; `pointProbeToSupportPod` is, imho, better.

Reply:
Much better. Done.